lang/french.json

```diff
@@ -236,19 +256,174 @@
+    "sydneyhud_hudlist_show_player_actions_desc" : "Décocher pour désactiver l'affichage de toutes les actions effectuées (Si activé, les actions individuelles peuvent être activés/désactivés ci-dessous)",
+    "sydneyhud_hudlist_not_ignore_item_crowbar" : "Pieds de biche",
+    "sydneyhud_hudlist_not_ignore_item_courier" : "Paquet de coursier de Gages",
+    "sydneyhud_hudlist_not_ignore_item_keycard" : "Carte-clé",
+    "sydneyhud_hudlist_not_ignore_item_meth_ingredients" : "Ingrédients de Meth",
+    "sydneyhud_hudlist_not_ignore_item_planks" : "Planches",
+    "sydneyhud_hudlist_not_ignore_item_secret_item" : "Items secrets",
+    "sydneyhud_hudlist_not_ignore_item_secret_item_desc" : "Différents items spéciaux tel que les bagues dans Birth of Sky et les bouteilles sur le Biker Heist",
+    "sydneyhud_hudlist_options_buff_options_buff" : "Options de buff",
+    "sydneyhud_hudlist_options_buff_options_buff_desc" : "Active/désactive des buffs spécifiques (Si Afficher les buffs est activé)",
+    "sydneyhud_hudlist_options_buff_options_composite" : "Buffs/effets composites",
+    "sydneyhud_hudlist_options_buff_options_boosts" : "Boosts",
+    "sydneyhud_hudlist_options_buff_options_boosts_desc" : "Effets de boost de Gage/IA",
+    "sydneyhud_hudlist_not_ignore_buff_combat_medic_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_forced_friendship_desc" : "Buff d'équipe",
+    "sydneyhud_hudlist_not_ignore_buff_hostage_taker_desc" : "Aussi inclus dans le buff composite de regénération de vie",
+    "sydneyhud_hudlist_not_ignore_buff_inspire_boost_cooldown" : "Temps de recharge deu boost de la motivation",
+    "sydneyhud_hudlist_not_ignore_buff_inspire_revive_cooldown" : "Temps de recharge de la réanimation de la motivation",
+    "sydneyhud_hudlist_not_ignore_buff_painkillers_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_quick_fix_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown" : "Temps de recharge des remontants",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown_desc" : "Les debuffs composite, s'afficheront en tant que qu'une partie de l'icone principale de buff même désactivé",
+    "sydneyhud_hudlist_not_ignore_buff_bullseye_cooldown" : "Temps de recharge de Dans l'mille",
+    "sydneyhud_hudlist_not_ignore_buff_die_hard_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_overkill_desc" : "Aussi inclus dans le buff composite de boost de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_shock_and_awe_desc" : "Buff d'équipe",
+    "sydneyhud_hudlist_not_ignore_buff_underdog_desc" : "Aussi inclus dans le buff composite de réduction de dégats et de boost de dégats des armes",
+    "sydneyhud_hudlist_not_ignore_buff_unseen_strike_cooldown" : "Unseen Strike Cooldown",
```

Comment:
Somehow you missed this one.
Unseen Strike in French is "Frappe Invisible"

Reply:
Yep, Indeed. Fixed